sdk/include/xdk/wdm.template.h

```diff
@@ -252,6 +252,7 @@ inline int IsEqualGUIDAligned(REFGUID guid1, REFGUID guid2)
 $include (wmitypes.h)
 
 $include (kefuncs.h)
+$include (kdfuncs.h)
```

Comment:
Why is this needed ?

Reply:
Using DbgPrint() before it was defined due to the previous placement.